utils/registry.py

```diff
@@ -79,7 +79,7 @@ def dispatch(self, input_keys: List[str], clump: bool = False):
                     successes += 1
         
         logger.info(
-            "Processing completed with '%s' successes, '%s' failures, and %s skipped.",
+            "Processing completed with '%s' successes and '%s' failures.",
```

Comment:
I think we should add skipped back in. I realize there was a bug on the following line where skipped wasn't included as an argument to the logger